### PR TITLE
mp b2b agent-customer-search

### PR DESCRIPTION
**Pull request by himanshu-saxena-metacube**
## PR Description

CC-16092 mp b2b agent customer search

### CODE REVIEW

File: tests/api/mp_b2b/glue/agent_endpoints/agent_customer_search/positive.robot [hunk 1]
```diff
@@ -0,0 +1,157 @@
+*** Settings ***
+Suite Setup       SuiteSetup
+Resource    ../../../../../../resources/common/common_api.robot
+Test Setup        TestSetup
+Default Tags    glue
+
+*** Test Cases ***
+ENABLER
+    TestSetup
+Agent_can_get_search_for_customers_without_search_parameters
```
Comment (OKrivtsova) [inline tests/api/mp_b2b/glue/agent_endpoints/agent_customer_search/positive.robot:10]: tests fail on staging, please fix
image

Comment (himanshu-saxena-metacube) [reply tests/api/mp_b2b/glue/agent_endpoints/agent_customer_search/positive.robot:10]: Fixed, Please check again

File: tests/api/mp_b2b/glue/agent_endpoints/agent_customer_search/positive.robot [hunk 2]
```diff
@@ -40,7 +40,7 @@ Agent_can_get_search_for_customers_by_first_name
     And Response header parameter should be:    Content-Type    ${default_header_content_type}
     And Response body parameter should be:    [data][0][id]    None
     And Response body parameter should be:    [data][0][type]    agent-customer-search
-    And Response should contain the array of a certain size:    [data][0][attributes][customers]    1
```
Comment (OKrivtsova) [inline tests/api/mp_b2b/glue/agent_endpoints/agent_customer_search/positive.robot]: why do we not check the size anymore?
we know how many are found we can check it

File: tests/api/mp_b2b/glue/agent_endpoints/agent_customer_search/positive.robot [hunk 3]
```diff
@@ -100,7 +100,6 @@ Agent_can_get_search_for_customers_by_substring
     And Response header parameter should be:    Content-Type    ${default_header_content_type}
     And Response body parameter should be:    [data][0][id]    None
     And Response body parameter should be:    [data][0][type]    agent-customer-search
-    And Response should contain the array of a certain size:    [data][0][attributes][customers]    2
```
Comment (OKrivtsova) [inline tests/api/mp_b2b/glue/agent_endpoints/agent_customer_search/positive.robot]: why do we not check the size anymore?
we know how many are found we can check it

File: tests/api/mp_b2b/glue/agent_endpoints/agent_customer_search/positive.robot [hunk 4]
```diff
@@ -149,8 +148,7 @@ Agent_can_get_search_for_customers_from_last_page
     And Response header parameter should be:    Content-Type    ${default_header_content_type}
     And Response body parameter should be:    [data][0][id]    None
     And Response body parameter should be:    [data][0][type]    agent-customer-search
-    And Response should contain the array of a certain size:    [data][0][attributes][customers]    6
```
Comment (OKrivtsova) [inline tests/api/mp_b2b/glue/agent_endpoints/agent_customer_search/positive.robot]: why do we not check the size anymore?
we know how many are found we can check it

File: tests/api/mp_b2b/glue/agent_endpoints/agent_customer_search/positive.robot (hunk 2 above)
Comment (himanshu-saxena-metacube) [reply tests/api/mp_b2b/glue/agent_endpoints/agent_customer_search/positive.robot]: Fixed

File: tests/api/mp_b2b/glue/agent_endpoints/agent_customer_search/positive.robot (hunk 3 above)
Comment (himanshu-saxena-metacube) [reply tests/api/mp_b2b/glue/agent_endpoints/agent_customer_search/positive.robot]: Fixed

File: tests/api/mp_b2b/glue/agent_endpoints/agent_customer_search/positive.robot (hunk 4 above)
Comment (himanshu-saxena-metacube) [reply tests/api/mp_b2b/glue/agent_endpoints/agent_customer_search/positive.robot]: Fixed

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**